drivers/holykell_hpt604.json

```diff
@@ -0,0 +1,10 @@
+{
+    "common" : {
+        "words" : 1,
+        "datatype" : "uint",
+        "fncode" : 3
+    },
+    "fields" : {
+      "level" : {"register" : 2,"description" : "Fuel level","unit" : "m","multiplier" : 0.001}
```

Comment:
Can we keep the use of spaces consistent with normal JSON formatting? (e.g. no space to the left of a colon or comma, but a space to the right)

Reply:
Definitely, thanks for raising this! Just pushed another commit